src/nvim/ex_cmds.lua

```diff
@@ -3003,6 +3003,12 @@ return {
     func='ex_map',
   },
   {
+    command='fnew',
+    flags=bit.bor(BANG, FILE1, RANGE, NOTADR, EDITCMD, ARGOPT, TRLBAR),
+    addr_type=ADDR_LINES,
+    func='ex_newfwin',
+  },
```

Comment:
Ordering of the commands is important, see https://github.com/neovim/neovim/blob/master/src/nvim/ex_cmds_defs.h#L15-L39. Otherwise the generated array `cmdidxs` is not correct.